caiman_napari/main_offline_gui.py

```diff
@@ -193,24 +200,19 @@ def show_mcorr_params_gui(self):
         self.mcorr_gui = MCORRWidget(parent=self)
         self.mcorr_gui.show()
 
-    def temp_func(self):
-        print("show results")
-        item_gui = QtWidgets.QListWidgetItem = self.ui.listWidgetItems.currentItem()
-        uuid = item_gui.data(3)
-        #algo = self.dataframe.loc[self.dataframe['uuid'] == uuid, 'algo']
-        self.viewer.open(self.dataframe.loc[self.dataframe['uuid'] == uuid,'input_movie_path'])
-        #print(self.dataframe.loc[self.dataframe['uuid'] == uuid])
-        load_outputs(self.viewer, self.dataframe.loc[self.dataframe['uuid'] == uuid])
     def load_output(self):
-        print("show results")
+        # clear napari viewer before loading new movies
+        self.clear_viewer()
+        # Find uuid for selected item
         item_gui = QtWidgets.QListWidgetItem = self.ui.listWidgetItems.currentItem()
         uuid = item_gui.data(3)
+        # Algorithm name for selected item
+        algo = self.dataframe.loc[self.dataframe['uuid'] == uuid, 'algo'].item()
+        # Open input movie for selected item
+        self.viewer.open(self.dataframe.loc[self.dataframe['uuid'] == uuid, 'input_movie_path'].item())
```

Comment:
I just realized this will not work if the `input_movie_path` is a `mmap` file. To solve this we should:

Split `main_offline_gui.open_movie()` into two functions. One which is the user-end function with the decorator and another private function `_open_movie()` which does the actual movie opening handling. `load_output()` here can then call the private function `_open_movie()` to open the movie file regardless of whether it's a mmap or other file type.